web/scripts/generate-index-files.ts

```diff
@@ -13,13 +13,13 @@ const allPaths = [...generalPaths, ...zonePaths];
 let filesGenerated = 0;
```

Comment:
This file is deleted :)

Reply:
I'll update the PR to reflect the current state 👍🏼